sdcm/utils/aws_region.py

```diff
@@ -551,3 +551,184 @@ def configure(self):
         self.create_sct_ssh_security_group()
         self.create_sct_key_pair()
         LOGGER.info("Region configured successfully.")
+
+
+class AwsRegion(AwsRegionBase):
```

Comment:
not sure I really understand the hierarchy here

it's hard to understand what are the changes like that, cause the code it copied